src/Navigator.tsx

```diff
@@ -156,11 +156,11 @@ const Navigator = (props: NavigatorProps) => {
             style={styles.root}>
             <Drawer.Navigator
               initialRouteName={initialRoute}
-              defaultStatus="closed"
-              drawerType={isLargeScreen ? 'permanent' : 'front'}
-              drawerStyle={
-                isLargeScreen ? {width: DEFAULT_FIXED_DRAWER_WIDTH} : undefined
-              }
+              screenOptions={{
```

Comment:
`drawerType={isLargeScreen ? 'permanent' : 'front'}` is this not supported in the latest version? this is for if user use tablet in landscape mode, the drawer will be permanent